tests/trino/test.sh

```diff
@@ -11,6 +11,7 @@ declare -A testCases=(
     [exchange_manager_values]="--values test-exchange-manager-values.yaml"
     [graceful_shutdown]="--values test-graceful-shutdown-values.yaml"
     [resource_groups_properties]="--values test-resource-groups-properties-values.yaml"
+    [session_properties]="--values test-session-properties-values.yaml"
```

Comment:
I don't think we need a new test, this could be added to `complete_values`, since we just check if the chart will render correctly.